pkg/local_workflows/code_workflow/native_workflow.go

```diff
@@ -55,12 +55,17 @@ func EntryPointNative(invocationCtx workflow.InvocationContext, opts ...Optional
 		return nil, err
 	}
 
-	summary := sarif2.CreateCodeSummary(&result.Sarif)
+	summary := sarif2.CreateCodeSummary(&result.Sarif, sarif2.WithCoverage(result.Coverage))
```

Comment:
Question: Is `result.Coverage` the correct field? There is also `runs[].properties.coverage`.

Reply:
After reviewing the SarifDocument we found the data we need in there. Refactor this to use the SarifDocument > Runs property.